pyseir/inference/model_fitter.py

```diff
@@ -266,9 +280,10 @@ def calculate_observation_errors(self):
         else:
             hosp_stdev = None
 
-        # Zero inflated poisson Avoid floating point errors..
-        cases_stdev[cases_stdev == 0] = 1e10
-        deaths_stdev[deaths_stdev == 0] = 1e10
+        # Zero inflated poisson Avoid floating point errors. This is set to a
+        # value that still provides some constraints toward zero.
```

Comment:
Minor: I don't understand the `Avoid floating point errors` statement, but I don't have a ton of context, and it is by no means a blocker

Reply:
old comment. cleaned up.